median/plugin.go

```diff
@@ -124,7 +131,12 @@ type latestRoundRequested struct {
 func (c *contractReaderContract) LatestTransmissionDetails(ctx context.Context) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, latestAnswer *big.Int, latestTimestamp time.Time, err error) {
 	var resp latestTransmissionDetailsResponse
 
-	err = c.contractReader.GetLatestValue(ctx, contractName, "LatestTransmissionDetails", primitives.Unconfirmed, nil, &resp)
+	binding := types.BoundContract{
+		Address: c.contractID,
+		Name:    contractName,
+	}
```

Comment:
contractReaderContract should have a binding variable instead that is set in NewMedianFactory